### DISCUSSION TITLE
Adding Filebeat modules

### DISCUSSION THREAD

**Discussion by PatRussell87** [Security-Onion-Solutions/securityonion#5357] [Unsupported Versions] [answered]
I know that SO has recently added support for Filebeat modules and can see in the config file where they are enabled. My question is whether it is possible to add a module that is not listed. I am hoping to feed Palo Alto logs into SO and have them parsed but the panw module is not listed in the default config for Filebeats. If it is possible to add modules not included in the OOTB install, what's the best way to do this?

I'm running 2.3.70 in an air-gapped, distributed deployment (sensor-master-storage).

**Comment by weslambert** [comment, marked as answer]
It is possible to add a new module in the same format in the pillar, however, you'll have to manually load the associated ingest node pipelines if they are not already loaded.
